docs/SPIR-V.rst

```diff
@@ -1194,6 +1194,14 @@ runtime array of unsigned integers, ``OpArrayLength`` is invoked on the runtime
 in order to find the number of unsigned integers. This is then multiplied by 4 to find
 the number of bytes.
 
+``.Load()``, ``.Load2()``, ``.Load3()``, ``.Load4()``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Since ByteAddressBuffers are represented as a struct with one member that is a
+runtime array of unsigned integers, the SPIR-V ``OpAccessChain`` instruction is used with
+the offset passed to the function, and ``OpLoad`` is used to load a 32-bit unsigned integer.
```

Comment:
IIRC, we are not directly using the offset passed to the function since that offset is the byte address? Need to divide by 4 of that offset?

Reply:
yeah, By "offset" I meant the value that is already divided by 4. But I can now see that it's not clear from the text. I'll clear that up.